scio-google-cloud-platform/src/main/scala/com/spotify/scio/bigquery/syntax/SCollectionSyntax.scala

```diff
@@ -84,6 +88,22 @@ final class SCollectionTableRowOps[T <: TableRow](private val self: SCollection[
     val param = TableRowJsonWriteParam(numShards, compression)
     self.covary[TableRow].write(TableRowJsonIO(path))(param)
   }
+
+  /**
+   * Save this SCollection to partitioned BigQuery tables using the specified table function.
+   * Note that elements must be of type
+   * [[com.google.api.services.bigquery.model.TableRow TableRow]].
+   */
+  def saveAsDynamicBigQueryTable(
```

Comment:
Sorry if I wasn't clear before I think these should be:
```suggestion
  def saveAsBigQueryTable(
```